### PR TITLE
Fix NXP-18852 NPE studio project uninstall

### PR DESCRIPTION
**Pull request by ChevalierFrugier**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nuxeo/nuxeo/pull/158" target="_blank">ReviewNinja

**Comment by kevinleturc** [comment]
Please edit your commit message with the NXP as prefix.

File: nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java [hunk 1]
```diff
@@ -244,7 +244,7 @@ public String getCurrentContentViewName() {
     }
 
     public void setCurrentContentViewName(String contentViewName) {
-        this.currentContentViewName = contentViewName;
```
Comment (kevinleturc) [inline nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java]: Is it needed to remove 'this' ?

File: nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java [hunk 2]
```diff
@@ -270,7 +270,10 @@ public void setCurrentSelectedSavedSearchId(String selectedSavedSearchId) throws
     protected void resetCurrentContentViewWorkingList() {
         if (currentContentViewName != null) {
             ContentView contentView = contentViewActions.getContentView(currentContentViewName);
-            documentsListsManager.resetWorkingList(contentView.getSelectionListName());
+            if(contentView != null){
```
Comment (kevinleturc) [inline nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java]: format
there's a space between if and (

**Comment by kevinleturc** [comment]
👍

File: nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java [hunk 3]
```diff
@@ -270,7 +270,10 @@ public void setCurrentSelectedSavedSearchId(String selectedSavedSearchId) throws
     protected void resetCurrentContentViewWorkingList() {
         if (currentContentViewName != null) {
             ContentView contentView = contentViewActions.getContentView(currentContentViewName);
-            documentsListsManager.resetWorkingList(contentView.getSelectionListName());
+            if (contentView != null) {
+                String selectionListName = contentView.getSelectionListName();
```
Comment (troger) [inline nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java]: Not sure we need to split it if we don't check something on `selectionListName `.

**Comment by ataillefer** [comment]
👍

File: nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java (hunk 3 above)
Comment (ChevalierFrugier) [inline nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java]: True, I could have put that in one line like documentsListsManager.resetWorkingList(contentView.getSelectionListName());
Should I correct the commit or just remember it for the next time ?

Comment (troger) [inline nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java]: Fix it :)

Comment (ChevalierFrugier) [inline nuxeo-features/nuxeo-search/nuxeo-search-ui/src/main/java/org/nuxeo/search/ui/seam/SearchUIActions.java]: Done :) !completed

**Pull request merged**